docs/content/preview/develop/change-data-capture/using-logical-replication/monitor.md

```diff
@@ -49,6 +49,7 @@ Contains mapping between publications and tables. It is a wrapper over `pg_publi
 | pubname | name | Name of publication. |
 | schemaname | name | Name of schema containing table. |
 | tablename | name | Name of table. |
+| attnames | name[] | Names of table columns included in the publication. This contains all the columns of the table when the user didn't specify the column list for the table. |
```

Comment:
Do we populate this correctly? Also we don't support specifying a column list to publication for streaming, so we should always get all the columns here.